extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -414,114 +409,112 @@ private function getBadgeMsgKeyPart($badgeTypeId) {
 		*/
 	}
 
-	public function getBadgeNameKey($badgeTypeId, $badge_lap = null) {
-		if($badgeTypeId == BADGE_LUCKYEDIT) {
+	public function getBadgeNameKey( $badgeTypeId, $badge_lap = null ) {
+		if( $badgeTypeId == BADGE_LUCKYEDIT ) {
 			$badge_lap = null;
 		}
-		return 'achievements-badge-name-'.$this->getBadgeMsgKeyPart($badgeTypeId) . (($badge_lap !== null) ? "-{$badge_lap}" : null);
+		return 'achievements-badge-name-' . $this->getBadgeMsgKeyPart( $badgeTypeId ) . ( ( $badge_lap !== null ) ? "-{$badge_lap}" : null );
 	}
 
-	public function getBadgeDescKey($badgeTypeId) {
-		if($this->getBadgeType($badgeTypeId) == BADGE_TYPE_INTRACKEDITPLUSCATEGORY)
+	public function getBadgeDescKey( $badgeTypeId ) {
+		if( $this->getBadgeType( $badgeTypeId ) == BADGE_TYPE_INTRACKEDITPLUSCATEGORY )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)